CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
 # Changelog
 
-## [Unreleased](https://github.com/ckan/ckanext-dcat/compare/v1.0.0...HEAD)
+## [Unreleased](https://github.com/ckan/ckanext-dcat/compare/v1.1.0...HEAD)
 
 * TBD
 
-## [v1.0.0](https://github.com/ckan/ckanext-dcat/compare/v0.0.9...v1.0.0)
+## [v1.1.0](https://github.com/ckan/ckanext-dcat/compare/v1.0.0...v1.1.0)
+
+* requirements.txt are Python 3 now (#167)
```

Comment:
This relates to the core requirements.txt not the extension one so I suggest we remove this line to avoid confusion